src/domains/ecommerce-common/00_alerts.tf

```
@@ -254,4 +254,49 @@ AzureDiagnostics
     operator  = "GreaterThanOrEqual"
     threshold = 2
   }
+}
+
+# eCommerce user stats service availability
+resource "azurerm_monitor_scheduled_query_rules_alert" "ecommerce_user_stats_service_availability_alert" {
+  count = var.env_short == "p" ? 1 : 0
+
+  name                = "ecommerce-user-stats-service-availability-alert"
+  resource_group_name = azurerm_resource_group.rg_ecommerce_alerts[0].name
+  location            = var.location
+
+  action {
+    action_group           = [data.azurerm_monitor_action_group.email.id, data.azurerm_monitor_action_group.slack.id]
+    email_subject          = "[eCommerce] User stats service availability less that 99%"
+    custom_webhook_payload = "{}"
+  }
+  data_source_id = data.azurerm_api_management.apim.id
+  description    = "eCommerce User stats service availability less than 98% in the last 30 minutes detected"
+  enabled        = true
+  query = (<<-QUERY
+let thresholdTrafficMin = 200;
+let thresholdTrafficLinear = 500;
+let lowTrafficAvailability = 94;
+let highTrafficAvailability = 98;
+let thresholdDelta = thresholdTrafficLinear - thresholdTrafficMin;
+let availabilityDelta = highTrafficAvailability - lowTrafficAvailability;
+AzureDiagnostics
+| where url_s startswith 'https://api.dev.platform.pagopa.it/ecommerce/user-stats-service/v1'
```

```suggestion
| where url_s startswith 'https://api.dev.platform.pagopa.it/ecommerce/user-stats-service/v1'
```
| where url_s startswith 'https://api.platform.pagopa.it/ecommerce/user-stats-service/v1'